### PR TITLE
Improve pdfcommand: 1. cleanup html files 2. able to set error handling

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: src/docfx/Models/PdfCommandOptions.cs [hunk 1]
```diff
@@ -20,6 +20,15 @@ internal class PdfCommandOptions : BuildCommandOptions
         [Option("generatesExternalLink", HelpText = "Specify whether or not generate external links for PDF")]
         public bool? GeneratesExternalLink { get; set; }
 
+        [Option("keepRawFiles", HelpText = "Specify whether or not keep the intermediate html files that used to generate the PDF file. It it usually used in debug purpose. By default the value is false")]
```
Comment (hellosnow) [inline src/docfx/Models/PdfCommandOptions.cs]: >whether or not keep [](start = 52, length = 19)

minor: whether or not 'to' keep?

File: src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs [hunk 2]
```diff
@@ -46,161 +46,171 @@ public static void PrerequisiteCheck()
...
         public void Convert()
         {
-            var fullPathTmpFolder = Path.Combine(_pdfOptions.SourceDirectory, TmpFolder);
-            if (Directory.Exists(fullPathTmpFolder))
+            var basePath = Path.GetFullPath(_pdfOptions.SourceDirectory);
+            try
             {
-                FolderUtility.ForceDeleteDirectoryWithAllSubDirectories(fullPathTmpFolder);
+                
+                ConvertCore(basePath);
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs]: >ConvertCore [](start = 16, length = 11)

remove above empty line

File: src/docfx/Models/PdfCommandOptions.cs [hunk 3]
```diff
@@ -20,6 +20,15 @@ internal class PdfCommandOptions : BuildCommandOptions
...
+        public bool? KeepRawFiles { get; set; }
+
+        [Option("errorHandling", HelpText = "Specify how to handle pdf pages that fail to load: abort, ignore or skip(default abort), it is the same input as wkhtmltopdf --load-error-handling options")]
+        public string LoadErrorHandling { get; set; }
+
+        [Option("rawOutputFolder", HelpText = "Specify the output folder for the raw files, if not specified, raw files will by default save to _raw subfolder under output folder if keepRawFiles is set to true.")]
```
Comment (superyyrrzz) [inline src/docfx/Models/PdfCommandOptions.cs]: >save [](start = 136, length = 4)

be saved

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**